tests/sentry/issues/test_attributes.py

```diff
@@ -237,14 +235,7 @@ def run_attr_test(
             }
             request = json_to_snql(json_body, "group_attributes")
             request.validate()
-            identity = lambda x: x
-            isolation_scope = sentry_sdk.Scope.get_isolation_scope().fork()
-            current_scope = sentry_sdk.Scope.get_current_scope().fork()
-            resp = _snuba_query(
-                ((request, identity, identity), isolation_scope, current_scope, {}, "test_api")
-            )
-            assert resp[0].status == 200
-            stuff = json.loads(resp[0].data)
+            stuff = raw_snql_query(request)
```

Comment:
... `stuff`, I know you're reusing the existing variable names but what the heck

Reply:
Good point, let me rename all these to something more meaningful since I'm here anyways